src/github/queries.gql

```diff
@@ -369,4 +379,27 @@ query GetMentionableUsers($owner: String!, $name: String!, $first: Int!, $after:
 		remaining
 		resetAt
 	}
-}
\ No newline at end of file
+}
+

```

Comment:
can you also update the `enterprise.gql` file? this query and the above should work with the oldest supported github enterprise version https://developer.github.com/enterprise/2.16/v4/object/repository/

Reply:
Checked it yesterday to see if it has support and missed/failed somehow